assets/src/blocks/Media/MediaEditor.js

```diff
@@ -0,0 +1,155 @@
+import { Fragment, useEffect, useState, useCallback } from "@wordpress/element";
+import { Tooltip, PanelBody } from '@wordpress/components';
+import { MediaPlaceholder } from "@wordpress/editor";
+import { InspectorControls } from '@wordpress/block-editor';
+
+import { URLInput } from "../../components/URLInput/URLInput";
+import { MediaEmbedPreview } from "./MediaEmbedPreview";
+import { MediaElementVideo } from './MediaElementVideo';
+import { useSelect } from '@wordpress/data';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+
+const renderEdit = ({ attributes, setAttributes }) => {
+  // Using a state to prevent the input losing the cursor position, a React issue reported multiple times
+  const [ mediaURL, setMediaURL ] = useState(attributes.youtube_id);
+  const debouncedMediaURLUpdate = useCallback(debounce(value => setAttributes({ youtube_id: value }), 300), []);
+
+  return (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+          <URLInput
+            label={__('Media URL/ID', 'planet4-blocks-backend')}
+            placeholder={__('Enter URL', 'planet4-blocks-backend')}
+            value={ mediaURL }
+            onChange={
+              value => {
+                setMediaURL(value)
+                debouncedMediaURLUpdate(value)
+              }
+            }
+            help={__('Can be a YouTube, Vimeo or Soundcloud URL or an mp4, mp3 or wav file URL.', 'planet4-blocks-backend')}
+          />
+        </PanelBody>
+      </InspectorControls>
+    </Fragment>
+  );
+}
+
+const renderView = (props, toAttribute) => {
+  const { attributes, setAttributes } = props;
+  const { video_title, description, embedHtml, youtube_id, posterURL } = attributes;
+
+  function onSelectImage({id}) {
+    setAttributes({video_poster_img: id});
+  }
+
+  function onSelectURL({url}) {
+    setAttributes({id: null});
+  }
+
+  function onUploadError({message}) {
+    console.log(message);
+  }
+
+  return (
+    <Fragment>
+      <Tooltip text={__('Edit text', 'planet4-blocks-backend')}>
```

Comment:
I'm not sure that we need a tooltip here 🤔 We added one for the Articles block due to the default value for the title (therefore it was not really clear that it was editable), here since it will be empty at first the placeholder should be enough! May be good to double check with Magali 🙂

Reply:
Ok, I'll check!